tests/test_misc.py

```diff
@@ -352,7 +228,7 @@ async def run_asyncio_loop(nursery, *, task_status=trio.TASK_STATUS_IGNORED):
                 # Trigger KeyboardInterrupt that should propagate accross the coroutines
                 signal.pthread_kill(threading.get_ident(), signal.SIGINT)
 
-
+@pytest.mark.xfail(reason="https://github.com/python-trio/trio-asyncio/issues/95", raises=RuntimeError)
```

Comment:
This change makes CI green. See also #95.

Reply:
Thanks for diagnosing this. I just uploaded #96 which appears to fix it. Probably worth incorporating into the release?